cli/upload.py

```diff
@@ -123,46 +123,63 @@ def _open_file_mapping(extra_metadata: dict, base_path: str) -> Dict[str, Binary
 )
 
 
-def _start_procs(upload_info: api.UploadInfo, xlsx: str) -> list:
+def _start_procs(srs_dst_pairs: list) -> list:
     """
     Starts multiple "gsutil cp" subprocesses.
     Returns a list of subprocess.Popen objects
```

Comment:
This docstring no longer accurately describes what this function does, since now there's fancy batching stuff going on.